SmartDeviceLink/public/SDLGetVehicleData.m

```diff
@@ -356,6 +362,14 @@ - (void)setHandsOffSteering:(nullable NSNumber<SDLBool> *)handsOffSteering {
     return [self.parameters sdl_objectForName:SDLRPCParameterNameHandsOffSteering ofClass:NSNumber.class error:nil];
 }
 
+- (void)setSeatOccupancy:(nullable NSNumber<SDLBool> *)seatOccupancy {
+    [self.parameters sdl_setObject:seatOccupancy forName:SDLRPCParameterNameSeatOccupancy];
+}
+
+- (nullable NSNumber<SDLBool> *)seatOccupancy {
+    return [self.parameters sdl_objectForName:SDLRPCParameterNameSeatOccupancy ofClass:NSNumber.class error:nil];
+}
+
 @end
```

Comment:
should this be of type `SeatOccupancy`? According to [spec](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0262-New-vehicle-data-SeatOccupancy.md#add-the-following-parameter-to-these-function-responses-1)